pkg/reconciler/revision/resources/queue_test.go

```diff
@@ -1122,7 +1142,8 @@ func probeJSON(container *corev1.Container) string {
 		return fmt.Sprintf(testProbeJSONTemplate, v1alpha1.DefaultUserPort)
 	}
 
-	if ports := container.Ports; len(ports) > 0 && ports[0].ContainerPort != 0 {
+	ports := container.Ports
+	if len(ports) > 0 && ports[0].ContainerPort != 0 {
```

Comment:
Why this change?

Reply:
probably a conflict resolution creep. fixed